packages/design-system-docs/src/pages/components/DatePicker/_CalendarPickerDateField.docs.scss

```diff
@@ -0,0 +1,50 @@
+/*
+Date field (with calendar picker)
+
+@uswds https://designsystem.digital.gov/components/date-picker/
+
+Date field (with calendar picker), also known as _date picker_, provides a single date entry with a text field or selection through a visual calendar element tied to the input.
+
```

Comment:
I think in this introductory section at the top we should explain how the differentiator is the `to*` and `from*` props. In my latest branch, I've changed it to require both a `to*` and `from*` prop in order to turn it into a picker.